src/utils/shareTypes.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Share types
+ */
+export const shareTypes = {
```

Comment:
One tiny thing that we did wrong in web: IMO this const should be named `shareType` (singular), so that you can use it as `shareType.link`, which feels more object oriented (and will happen in the future). Could you introduce it here as `shareType`? :-)